CONTRIBUTING.md

```diff
@@ -17,12 +17,36 @@ want a `.war` as quick as possible, you can use this instead:
 $ mvn -Dmaven.test.skip=true clean package 
 ```
 
+
 # Suggesting modifications
 
 Airsonic's source code is hosted on [github](https://github.com/airsonic/airsonic/),
 who provides a [lot of documentation](https://help.github.com/en) on how
 to contribute to projects hosted there.
 
+Keep in mind that we are a small project, and that we value stability over new
+features.
```

Comment:
I really don't understand this sentence?
Build stability? Your travis should take care of that?
What kind of stability are we talking about with this sentence?
Releases cycles should be an orthogonal concern from commits

Reply:
I rephrased this a bit :)